dist/releases/4.12/config.toml

```diff
@@ -175,6 +180,12 @@ filter_files = [
   "/usr/libexec/cni/tuning",
   "/usr/libexec/cni/vlan",
   "/usr/libexec/cni/vrf",
-  "/usr/libexec/podman/rootlessport",
 ]
 
+[[tag.rhel-coreos-8.ignore_errors]]
```

Comment:
Why can't this one by payload.rhel-coreos-8.ignore_errors now?

Reply:
If operator name would be `rhel-coreos-8`, it can be used now. The problem is, in e.g. 4.12, we have something like this:

| OPERATOR NAME                            | TAG NAME                   | EXECUTABLE NAME                               | STATUS                                                          | IMAGE                                                                                                                  |
|------------------------------------------|----------------------------|-----------------------------------------------|-----------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------|
|                                          | rhel-coreos-8              | /usr/bin/crio                                 | go binary does not contain required tag(s): [strictfipsruntime] | quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:5ef4276442c5174d31f6b62a83aa40e64c719275dd731e5ccb0dc98911f7e57e |

IOW we have some errors which does not have operator name set, only the tag name, so we have to have both per-operator name (`payload.*`) and per-tag name (`tag.*) exclusion lists.

cc @rphillips

Reply:
okay!

Reply:
I think it could be payload.* with either operator name or tag name for match. Right now with tag.* it isnt' clear that it is nested under a payload. (We can change this later).